src/main/kotlin/net/atos/zac/mail/MailjetService.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.mail
+
+import com.mailjet.client.ClientOptions
+import com.mailjet.client.MailjetClient
+
+object MailjetService {
+    fun createMailjetClient(mailjetApiKey: String, mailjetSecretKey: String): MailjetClient =
+        MailjetClient(
+            ClientOptions.builder()
+                .apiKey(mailjetApiKey)
+                .apiSecretKey(mailjetSecretKey)
+                .build()
+        )
+}
```

Comment:
No need for an object here. That's just overkill. A Kotlin object is a singleton in Java. Better to use pure functions where possible.

```suggestion
    fun createMailjetClient(mailjetApiKey: String, mailjetSecretKey: String): MailjetClient =
        MailjetClient(
            ClientOptions.builder()
                .apiKey(mailjetApiKey)
                .apiSecretKey(mailjetSecretKey)
                .build()
        )
```

Reply:
Sure, but how do we mock this in tests? That's the only purpose of this object/class at the moment

Reply:
Why would you want to mock such a static function? I don't see what using an object has to do with that really?

PS: you can mock static functions but I don't think you would ever want to mock this particular function?

Reply:
The word `Service` in `MailjetService` is also very misleading. It's more a helper. `MailjetUtil` or something?

Reply:
You can use this to mock pure Kotlin functions:

```  mockkStatic(::createMailjetClient)
    every {
        createMailjetClient("dummyApiKey", "dummySecretKey")
    } returns mailJetClient
    ```
    
    With this you can remove the `object`

Reply:
Works! Thanks @edgarvonk